libs/service-portal/documents/src/screens/Overview/Overview.tsx

```diff
@@ -155,6 +149,10 @@ export const ServicePortalDocuments: ServicePortalModuleComponent = ({
   const handleSearchChange = useCallback((value: string) => {
     setPage(1)
     setFilterValue({ ...defaultFilterValues, searchQuery: value })
+    if (!searchInteractionEventSent && pathname) {
+      documentsSearchDocumentsInitialized(pathname)
+      searchInteractionEventSent = true
```

Comment:
Use a state variable, this will probably run on every keydown because the variable will reinitialize to false on every render
You can also move useFilteredDocuments back out of the component